app/views/api/vacancies/index.json.jbuilder

```diff
@@ -1,3 +1,19 @@
-json.vacancies @vacancies.decorated_collection do |vacancy|
+json.info do
```

Comment:
Reading through https://jsonapi.org/format/#document-top-level should this be `meta`?

Reply:
No. We are following the Open API standard [as specified in the GDS guide](https://www.gov.uk/guidance/gds-api-technical-and-data-standards#document-your-api).

https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md

<img width="945" alt="screen_shot_2018-11-06_at_13_28_46_1" src="https://user-images.githubusercontent.com/159200/48067445-3bee9380-e1c8-11e8-81c2-4c6a97e8420b.png">

Reply:
👍 thanks for linking